go.sum

```diff
@@ -5,6 +5,9 @@ github.com/armon/go-metrics v0.0.0-20180917152333-f0300d1749da h1:8GUt8eRujhVEGZ
 github.com/armon/go-metrics v0.0.0-20180917152333-f0300d1749da/go.mod h1:Q73ZrmVTwzkszR9V5SSuryQ31EELlFMUz1kKyl939pY=
 github.com/armon/go-radix v0.0.0-20180808171621-7fddfc383310/go.mod h1:ufUuZ+zHj4x4TnLV4JWEpy2hxWSpsRywHrMgIH9cCH8=
 github.com/bgentry/speakeasy v0.1.0/go.mod h1:+zsyZBPWlz7T6j88CTgSN5bM796AkVf0kBD4zp0CCIs=
+github.com/cenkalti/backoff v1.1.0 h1:QnvVp8ikKCDWOsFheytRCoYWYPO/ObCTBGxT19Hc+yE=
+github.com/cenkalti/backoff v2.2.1+incompatible h1:tNowT99t7UNflLxfYYSlKYsBpXdEet03Pg2g16Swow4=
```

Comment:
You are importing versions 1 and 2 of the backoff library. I am thinking that you want to use v2, but go thinks that you want to use both because you reference `"github.com/cenkalti/backoff"` in your import, but explicitly have `github.com/cenkalti/backoff v2.2.1+incompatible` in your `go.mod`.

IIRC the solution is to change your import to `"github.com/cenkalti/backoff/v2"` and then run `go mod tidy`

Source [Semantic Import Versioning](https://research.swtch.com/vgo-import)

Reply:
I just realized this might be from a lib that you are using that is using the backoff v2. If that's the case, please ignore me. :)

Reply:
Thanks for the comments @guygrigsby. Indeed the tidy cleaned up a bit removing the v1.